annif/cli.py

```diff
@@ -345,35 +364,51 @@ def run_learn(project_id, paths, docs_limit, backend_param):
 
 @cli.command("suggest")
 @click.argument("project_id")
+@click.argument("paths", type=click.Path(dir_okay=False, exists=True), nargs=-1)
```

Comment:
Thought: Should it be possible to pass a directory (containing .txt documents) as well?